apps/vaporgui/PlotParams.cpp

```diff
@@ -40,7 +40,9 @@ static RenParamsRegistrar<PlotParams> registrar( PlotParams::GetClassType() );
 
 PlotParams::PlotParams( DataMgr* dmgr, ParamsBase::StateSave *ssave) 
           : RenderParams( dmgr, ssave, PlotParams::GetClassType()) 
-{ }
+{
+    SetUseSingleColor(true);
```

Comment:
How does this change to PlotParams fix a problem with the isosurface renderer?

Reply:
My bad git manipulations... Now it's at the right place.